### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Philipp-Kaiser**
Have a look on user story #3!

Added GlobalStyle, some basic structure (all empty) and the component navbar with react router.

### CODE REVIEW

**Comment by mve1983** [comment]
Navbar ist working fine, but there are two console log errors.

Invalid Dom Property: enable-background for svgs (see above)
Further the fetch in line 14 of App.jsx throws also console-error

all other lgtm. :)

File: client/src/index.css [hunk 1]
```diff
@@ -1,8 +1,96 @@
...
+  --white: #ecf8f8;
+  --black: #211103;
+
+  --ff-sans: sans-serif;
+  --ff-serif: serif;
+
+  --fs-200: 0.75rem;
+  --fs-300: 1rem;
+  --fs-400: 1.25rem;
+  --fs-500: 1.375rem;
```
Comment (anitaschiller) [inline client/src/index.css:17]: Are you sure that you need different/more variables here in the index.css although you already have some in the GlobalStyles?

**Review by anitaschiller — COMMENTED** [review]
Left one comment ;)

File: client/src/index.css (hunk 1 above)
Comment (Philipp-Kaiser) [reply client/src/index.css:17]: Totally forgot about the index.css.. 
Thanks for the review!

**Pull request merged**